go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/oapi-codegen/runtime v1.1.1
 	github.com/operator-framework/api v0.27.0
 	github.com/operator-framework/operator-lifecycle-manager v0.27.0
-	github.com/percona/everest-operator v0.6.0-dev1.0.20250408103821-238475bf8caa
+	github.com/percona/everest-operator v0.6.0-dev1.0.20250415133039-c52283125953
```

Comment:
Update to version from main after merging the everest operator PR

Reply:
done